make/langtools/tools/compileproperties/CompileProperties.java

```diff
@@ -249,7 +249,7 @@ private boolean createFile(String propertiesPath, String outputPath,
             Writer writer = null;
             try {
                 writer = new BufferedWriter(
-                        new OutputStreamWriter(new FileOutputStream(outputPath), StandardCharsets.UTF_8));
+                        new OutputStreamWriter(new FileOutputStream(outputPath), StandardCharsets.ISO_8859_1));
```

Comment:
Using ISO_8859_1 seems strange.
Since these are generated files, you could write them as UTF-8 and then override the default javac option for ascii when compiling _just_ these files.

Reply:
Will stick with your latter solution, as since the .properties files were converted via native2ascii, it makes sense to write out via ascii.